src/pydicom/filereader.py

```diff
@@ -242,14 +242,7 @@ def data_element_generator(
                     dotdot = "..." if length > 20 else "   "
                     displayed_value = value[:20] if value else b""
                     logger_debug(
-                        "%08x: %-34s %s %r %s"
-                        % (
-                            value_tell,
-                            bytes2hex(displayed_value),
-                            dotdot,
-                            displayed_value,
-                            dotdot,
-                        )
+                        f"{value_tell:08x}: {bytes2hex(displayed_value):<34} {dotdot} {displayed_value!r} {dotdot}"
```

Comment:
Much nicer in f-string ... I had thought we had found and replaced all of these already ... I guess not.